microsoft-edge/webview2/concepts/versioning.md

```diff
@@ -3,89 +3,152 @@ description: Version Models used for Microsoft Edge WebView2
+*  The set of APIs in the WebView2 General Availability release (build 616) requires that the client has WebView2 Runtime version `86.0.616.0` or newer.
+*  Full support for the latest APIs in an SDK release requires that the Runtime on the client have a version number greater than or equal to the SDK version.
 
-### Roadmap  
 
-The release package contains all of the stable, supported Win32 C/C++ and .NET APIs.  The prerelease package contains experimental APIs that are subject to change based on your feedback.  
+## Release and prerelease packages
 
+The NuGet package for the WebView2 SDK contains both a release and prerelease package.
+
+*  The _release_ package of the SDK contains all of the stable Win32 C/C++ and .NET APIs.  APIs that are in the release package are fully supported.  The release package is forward-compatible and contains the following components:
+    *  [Win32 C/C++ APIs][ReferenceWin32].
+    *  .NET APIs:  [WPF][DotnetMicrosoftWebWebview2WpfNamespace], [WinForms][DotnetMicrosoftWebWebview2WinformsNamespace], and [Core][DotnetMicrosoftWebWebview2CoreNamespace].  
+
+*  The _prerelease_ package of the SDK is a superset of the release package, with additional, [Experimental APIs](#experimental-apis) that are subject to change based on your feedback.  Avoid using the prerelease version of the WebView2 SDK to build production apps.
+
+
+<!-- possibly condense section, having added para3 -->
 ## Experimental APIs  
 
-The WebView team is seeking feedback on experimental APIs that may be included in future releases.  The experimental APIs are marked as `experimental` in the SDK.  To help you evaluate the Experimental APIs and share your feedback, navigate to the [WebView feedback repo][GithubMicrosoftedgeWebviewfeedback].  
+<!-- from paragraph 3 of post from github issue 341: -->
+### Experimental APIs are not guaranteed to be forward-compatible
+
+Any experimental APIs in a pre-release SDK are not guaranteed to be forward-compatible.   When a _pre-release_ version of the WebView2 SDK is initially made available, that SDK only works with Microsoft Edge Canary.  After that, the SDK works with the Beta and Dev channels, for the development phase of your app.  A pre-release version of the WebView2 SDK is meant for developers to try out new APIs early and provide feedback before the new APIs are promoted to become stable, forward-compatible APIs.
+
+For a release version of your app, use a _Release_ version of the WebView2 SDK.  The _stable_ (non-experimental) APIs that are in that SDK version are forward-compatible.  
+
+The WebView2 SDK has been forward-compatible ever since version 1.  If your WebView2 app uses the current version of the SDK, your app continues to work even after the client's WebView2 Runtime is updated.  However, any experimental APIs that are in a version of the WebView2 SDK are not guaranteed to be forward-compatible.
```

Comment:
```suggestion
The WebView2 _release_ SDK has been forward-compatible ever since version 1.  If your WebView2 app uses the current version of the SDK, your app continues to work even after the client's WebView2 Runtime is updated.  However, any experimental APIs that are in a version of the WebView2 SDK are not guaranteed to be forward-compatible.
```